resources/lang/en/beatmap_discussions.php

```diff
@@ -78,4 +78,11 @@
         'everyone' => 'Everyone',
         'label' => 'Filter by user',
     ],
+
+    'user_title' => [
```

Comment:
missing all the other groups
https://github.com/ppy/osu-web/blob/8e2da62880bda8a163265b410666bf06a11da0d8/resources/assets/coffee/osu_common.coffee#L257